nxcomp/src/Proxy.cpp

```diff
@@ -6294,7 +6294,11 @@ int Proxy::handleNewGenericConnectionFromProxyUnix(int channelId, T_channel_type
 
   serverAddrUnix.sun_family = AF_UNIX;
 
-  const int serverAddrNameLength = 108;
+  // Some OSses define sockaddr_un.sun_path with a string length of 108 chars
+  // some to string length of 104 chars.
+  //
+  // Let's make them all happy by using the shorter string size...
+  const int serverAddrNameLength = 104;
```

Comment:
Good that I take another look at this. I totally missed that this section is for *general* UNIX sockets, not (just) launchd ones.

Let's better do

```C++
#ifdef __linux__
 ... = 108;
#else
  /* POSIX/SUS does not specify a length.
   * BSD derivatives generally support 104 bytes, other systems may be more constrained.
   * If you happen to run into such systems, extend this section with the appropriate limit.
   */
 ... = 104;
#endif